src/Runner.Common/HostContext.cs

```diff
@@ -94,6 +94,13 @@ public HostContext(string hostType, string logFile = null)
             this.SecretMasker.AddValueEncoder(ValueEncoders.PowerShellPreAmpersandEscape);
             this.SecretMasker.AddValueEncoder(ValueEncoders.PowerShellPostAmpersandEscape);
 
+            // Create StdoutTraceListener if ENV is set
+            StdoutTraceListener stdoutTraceListener = null;
+            if (StringUtil.ConvertToBoolean(Environment.GetEnvironmentVariable(Constants.Variables.Agent.PrintLogToStdout)))
+            {
+                stdoutTraceListener = new StdoutTraceListener();
```

Comment:
you might want to pass hostType to `StdoutTraceListener`, ex: `new StdoutTraceListener(hostType)`